compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -1338,7 +1339,7 @@ private boolean isTupleTypeAssignableToArrayType(BTupleType source, BArrayType t
     private boolean isArrayTypeAssignableToTupleType(BArrayType source, BTupleType target,
                                                      Set<TypePair> unresolvedTypes) {
         BType restType = target.restType;
-        List<BTupleMember> tupleTypes = target.members;
+        List<BTupleMember> tupleTypes = target.getMembers();
```

Comment:
Multiple occurrences in this class where we are using types to indicate members.